packages/arb-token-bridge-ui/src/util/deposits/fetchDepositsFromSubgraph.ts

```diff
@@ -49,13 +49,13 @@ export const fetchDepositsFromSubgraph = async ({
   sender?: string
   receiver?: string
   fromBlock: number
-  toBlock: number
+  toBlock?: number
```

Comment:
`toBlock` should be optional, because if not provided it should default to latest